floodviz/views.py

```diff
@@ -1,7 +1,41 @@
-from flask import request, render_template, abort
+import json
+import requests
+from flask import render_template
 
 from . import app
+from . import hydrograph_utils
+
 
 @app.route('/home/')
 def home():
-    return render_template('index.html')
\ No newline at end of file
+    # new data store #
+    newd = []
+    sites = app.config['SITE_IDS']
+    start_date = app.config['START_DT']
+    end_date = app.config['END_DT']
+    n_show_series = app.config['N_SERIES']
+    hydro_meta = app.config['HYDRO_META']
+    url_top = app.config['NWIS_SITE_SERVICE_ENDPOINT']
+    sites_value_maxes = {}
+
+    # Set up to retrieve all site ids from url? #
+    for idx, site in enumerate(sites):
+        url =  url_top +'iv/?site=' + site + '&startDT=' + \
+              start_date + '&endDT=' + end_date + '&parameterCD=00060&format=json'
+
+        r = requests.get(url)
+
+        if r.status_code is 200:
```

Comment:
We should think about what to do in case the status code is not 200. One easy thing to do would be to write the file out with an empty object. This may be what happens below. I'm not sure but it is something to consider and perhaps at least do a test where you get an error and see what happens in the application.